### DISCUSSION TITLE
How can I disable the "Would you like to generate java source files out of your proto piles" popup every time I open a project?

### DISCUSSION THREAD

**Discussion by kchristidis** [redhat-developer/vscode-java#3116] [Q&A]
Bonus points if I can have it default to generating the source files.

**Comment by kchristidis** [comment]
Bumping this up?

**Comment by snjeza** [comment]
@kchristidis Could you provide a project example?

**Comment by kchristidis** [comment]
Unfortunately, all of the repos that I'm working on are private. Do I take that to mean that the reported behavior is not common?

**Comment by snjeza** [comment]
Could you show your pom.xml or build.gradle?

**Comment by kchristidis** [comment]
Unfortunately, I cannot. I understand that I am not giving you enough to work with here, thus making it impossible for you to help me. I will close this thread. Thank you for offering to take a look.
